aviatrix_ha/handlers/cft/handler.py

```diff
@@ -0,0 +1,146 @@
+import os
+import traceback
+
+from aviatrix_ha.api.external.ami import check_ami_id
+from aviatrix_ha.csp.eip import is_ip_elastic
+from aviatrix_ha.csp.instance import verify_iam
+from aviatrix_ha.csp.lambda_c import set_environ, update_env_dict
+from aviatrix_ha.csp.s3 import (
+    MAXIMUM_BACKUP_AGE,
+    is_backup_file_is_recent,
+    verify_backup_file,
+    verify_bucket,
+)
+from aviatrix_ha.errors.exceptions import AvxError
+from aviatrix_ha.handlers.cft.create import setup_ha
+from aviatrix_ha.handlers.cft.delete import delete_resources
+from aviatrix_ha.handlers.cft.response import send_response
+
+
+def handle_cft(
+    describe_err,
+    event,
+    context,
+    client,
+    lambda_client,
+    controller_instanceobj,
+    instance_name,
+):
+    """Handle CFT event"""
+    if describe_err:
+        print("From CF Request")
+        if event.get("RequestType", None) == "Create":
+            print("Create Event")
+            send_response(event, context, "FAILED", describe_err)
+            return
+        print("Ignoring delete CFT for no Controller")
+        # While deleting cloud formation template, this lambda function
+        # will be called to delete AssignEIP resource. If the controller
+        # instance is not present, then cloud formation will be stuck
+        # in deletion.So just pass in that case.
+        send_response(event, context, "SUCCESS", "")
+        return
+
+    try:
+        response_status, err_reason = _handle_cloud_formation_request(
+            client, event, lambda_client, controller_instanceobj, context, instance_name
+        )
+    except AvxError as err:
+        err_reason = str(err)
+        print(err_reason)
+        response_status = "FAILED"
+    except Exception as err:  # pylint: disable=broad-except
+        err_reason = str(err)
+        print(traceback.format_exc())
+        response_status = "FAILED"
+
+        # Send response to CFT.
+    if response_status not in ["SUCCESS", "FAILED"]:
+        response_status = "FAILED"
+    send_response(event, context, response_status, err_reason)
+    print("Sent {} to CFT.".format(response_status))
+
+
+def _handle_cloud_formation_request(
```

Comment:
Same comment about breaking this up a bit. Again  feel free to ignore if your aim is not to clean this up now.